examples/fork-variable-access/fork_variable_access.metatags

```diff
@@ -0,0 +1,2 @@
+description: This sample demonstrates how the workers within a fork can access the variables that are declared in the scope of that block.
```

Comment:
```suggestion
This Ballerina sample demonstrates how the workers within a fork can access the variables that are declared in the scope of that block.
```

Reply:
Try to add the keywords like Ballerina in the description too. Add "Ballerina" before sample in all BBEs without the Ballerina keyword.